### PR TITLE
en, zh: add release notes for v1.2.0-rc.1

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

add release notes for v1.2.0-rc.1

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/releases/release-1.2.0-rc.1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- 支持为 Grafana 配置额外的 volumeMount ([#3960](https://github.com/pingcap/tidb-operator/pull/3960), [@mikechengwei](https://github.com/mikechengwei))
+- 为 TidbMonitor 增加额外的信息展示列 ([#3958](https://github.com/pingcap/tidb-operator/pull/3958), [@mikechengwei](https://github.com/mikechengwei))
+- TidbMonitor 支持将配置信息支持写入到 PD 的 etcd 中 ([#3924](https://github.com/pingcap/tidb-operator/pull/3924), [@mikechengwei](https://github.com/mikechengwei))
+
+## Bug 修复
+
+- 修复 TidbMonitor 可能无法对启用了 TLS 的 DmCluster 进行监控的问题 ([#3991](https://github.com/pingcap/tidb-operator/pull/3991), [@csuzhangxc](https://github.com/csuzhangxc))
+- 修复 PD 在扩容过程中 member 数量统计不正确的问题 ([#3940](https://github.com/pingcap/tidb-operator/pull/3940), [@cvvz](https://github.com/cvvz))
+- 修复 DM-master 可能无法重启成功的问题 ([#3972](https://github.com/pingcap/tidb-operator/pull/3972), [@csuzhangxc](https://github.com/csuzhangxc))
+- 修复将 `configUpdateStrategy` 从 `InPlace` 修改为 `RollingUpdate` 时可能造成 TidbCluster 组件滚动更新的问题 ([#3970](https://github.com/pingcap/tidb-operator/pull/3970), [@cvvz](https://github.com/cvvz))
```
Comment (qiancai) [inline zh/releases/release-1.2.0-rc.1.md:32]:
```suggestion
- 修复将 `configUpdateStrategy` 从 `InPlace` 修改为 `RollingUpdate` 后可能造成的 TidbCluster 组件滚动更新的问题 ([#3970](https://github.com/pingcap/tidb-operator/pull/3970), [@cvvz](https://github.com/cvvz))
```

**Review by qiancai — COMMENTED** [review]
@csuzhangxc  PTAL

File: en/releases/release-1.2.0-rc.1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+TiDB Operator version: 1.2.0-rc.1
+
+## Rolling update changes
+
+- Upgrading TiDB Operator will cause the recreation of the Pump Pod due to [#3973](https://github.com/pingcap/tidb-operator/pull/3973)
+
+## New features
+
+- Support customized labels for TidbCluster pods and services ([#3892](https://github.com/pingcap/tidb-operator/pull/3892), [@SabaPing](https://github.com/SabaPing) [@july2993](https://github.com/july2993))
```
Comment (qiancai) [inline en/releases/release-1.2.0-rc.1.md:17]:
```suggestion
- Support customized labels for TidbCluster Pods and services ([#3892](https://github.com/pingcap/tidb-operator/pull/3892), [@SabaPing](https://github.com/SabaPing) [@july2993](https://github.com/july2993))
```

**Review by qiancai — COMMENTED** [review]
@csuzhangxc  PTAL

File: en/releases/release-1.2.0-rc.1.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## New features
+
+- Support customized labels for TidbCluster Pods and services ([#3892](https://github.com/pingcap/tidb-operator/pull/3892), [@SabaPing](https://github.com/SabaPing) [@july2993](https://github.com/july2993))
+- Support full lifecycle management for Pump ([#3973](https://github.com/pingcap/tidb-operator/pull/3973), [@july2993](https://github.com/july2993))
+
+## Improvements
+
+- Mask the backup password in logging ([#3979](https://github.com/pingcap/tidb-operator/pull/3979), [@dveeden](https://github.com/dveeden))
+- Add an additional volumeMounts field for Grafana ([#3960](https://github.com/pingcap/tidb-operator/pull/3960), [@mikechengwei](https://github.com/mikechengwei))
+- Add some useful additional printout columns for TidbMonitor ([#3958](https://github.com/pingcap/tidb-operator/pull/3958), [@mikechengwei](https://github.com/mikechengwei))
```
Comment (csuzhangxc) [inline en/releases/release-1.2.0-rc.1.md:24]:
```suggestion
- Add several useful additional printout columns for TidbMonitor ([#3958](https://github.com/pingcap/tidb-operator/pull/3958), [@mikechengwei](https://github.com/mikechengwei))
```

File: zh/releases/release-1.2.0-rc.1.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+# TiDB Operator 1.2.0-rc.1 Release Notes
+
+发布日期：2021 年 5 月 28 日
+
+TiDB Operator 版本：1.2.0-rc.1
+
+## 滚动升级改动
+
+- 由于 [#3973](https://github.com/pingcap/tidb-operator/pull/3973) 的改动，升级 TiDB Operator 会导致 Pump Pod 删除重建。
```
Comment (qiancai) [inline zh/releases/release-1.2.0-rc.1.md:13]:
```suggestion
- 由于 [#3973](https://github.com/pingcap/tidb-operator/pull/3973) 的改动，升级 TiDB Operator 会导致 Pump Pod 删除重建
```

File: zh/releases/release-1.2.0-rc.1.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## 新功能
+
+- 支持为 TidbCluster 中的 Pod 与 service 设置自定义的 label ([#3892](https://github.com/pingcap/tidb-operator/pull/3892), [@SabaPing](https://github.com/SabaPing) [@july2993](https://github.com/july2993))
+- 完整支持对 Pump 的生命周期管理 ([#3973](https://github.com/pingcap/tidb-operator/pull/3973), [@july2993](https://github.com/july2993))
```
Comment (qiancai) [inline zh/releases/release-1.2.0-rc.1.md:18]:
```suggestion
- 支持对 Pump 的完整生命周期管理 ([#3973](https://github.com/pingcap/tidb-operator/pull/3973), [@july2993](https://github.com/july2993))
```

**Pull request merged**